Sample/sdk-ios-swift/ViewController.swift

```diff
@@ -11,127 +11,117 @@ import YotiButtonSDK
 
 class ViewController: UIViewController {
     
-    var responseObject: [String: Any]?
+    var responseObject:  ProfileDictionary?
+    @IBOutlet weak var rememberMeButton: YotiButton!
+    @IBOutlet weak var selfieAuthButton: YotiButton!
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        
-        do {
-            
-            guard let url = URL(string: "https://android-test-yoti.herokuapp.com/profile-json") else {
-                return
-            }
-            let scenario = try ScenarioBuilder().setUseCaseID("yoti_btn_1")
-                .setClientSDKID("d28feaf4-d62d-40e3-88ae-d619e9a5b906")
-                .setScenarioID("60b8e997-4a5c-40b2-86e8-29c4521b7015")
-                .setClientCompletion({ (baseURL, token, url, error) in
-                    print("token: \(String(describing: token))")
-                    print("url: \(String(describing: url))")
-                    // If you decide to call the callback url via
-                    // your own Service and handle the result, set  isProcessed at true.
-                    // Otherwise, set isProcessed at false and the Yoti SDK
-                    // will make the call.
-                    return false
-                })
-                .setCallbackBackendURL(url)
-                .setBackendCompletion({ (data, error) in
-                    print("=== Backend Completion ===")
-                    print("object: \(String(describing: data))")
-                    print("error: \(String(describing: error))")
-                    
-                    self.responseObject = try? JSONSerialization.jsonObject(with: data!, options: []) as! [String : Any]
-
-                    
-                    if error == nil {
-                        DispatchQueue.main.async {
-                            self.moveToProfile()
-                        }
-                    }
-                    
-                    
-                })
-                .create()
-            YotiSDK.add(scenario: scenario)
+    
+    @IBAction func yotiButtonDidTouchUpInside(_ sender: YotiButton) {
+        guard let useCaseID = sender.useCaseID else{
+             return
         }
-        catch {
-            
+        do {
+            UIApplication.shared.isNetworkActivityIndicatorVisible = true
+            try YotiSDK.startScenario(for: useCaseID, with: self)
+        } catch {
+            // FIXME
         }
-
-        // Do any additional setup after loading the view, typically from a nib.
     }
     
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        rememberMeButton.setTitle("RememberMe Scenario", for: .normal)
+        selfieAuthButton.setTitle("SelfieAuth Scenario", for: .normal)
```

Comment:
I think those two can be set into the storyboard.

Reply:
It doesn't seem to take the names from the storyboard. I think it is because we are setting the title "Use Yoti" already in the code of YotiButton class inside the SDK.  It works like this though.